### PR TITLE
CE for cloud version

### PR DESCRIPTION
**Pull request by gabrielciobotaru77**

### CODE REVIEW

File: cloud_config/enable_cloud.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#Concatenate the .cloud files, with the .env files required for docker compose
```
Comment (aphethean1) [inline cloud_config/enable_cloud.sh:1]: Rename `cloud_config/enable_cloud.sh` to `cloud_config/run.sh`

File: cloud_config/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Steps to run the cloud version:
+
+1. Generate the SSL certificates for app and catalog:
+    Option 1 - generate 2 certificates, one for app and one for catalog (company-app.domain.com and company-catalog.domain.com). Copy required files in the certificate appropriate folders
+    Option 2 - generate 1 certificate for both app and catalog (matatika.domain.com). In this case, please copy the required files into both location (duplicate the files) - in this case, ports for app and catalog can't be the same. Please change those in nginx.conf file included in this folder.
+2. Edit nginx.conf file and replace <fullpath> with the full path of the folders. Also edit the <app_dns_name> and <catalog_dns_name> with your dns names for app and catalog. 
+3. Edit the variables in .env.app.cloud file
+4. Edit the variables in .env.catalog.cloud file
+5. Edit enable_cloud.sh file <fullpath> for the nginx configuration file needs to be added. Nginx needs to point to this configuration file, and full path is required since. This shell script will copy the cloud configuration files to be loaded by docker compose, and also will start the Matatika CE and also start the nginx.
```
Comment (aphethean1) [inline cloud_config/README.md:19]: Update to run.sh

File: cloud_config/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+`app_certificate/fullchain.pem`: the certificate file used in most server software.  - for the application
+`catalog_certificate/privkey.pem`  : the private key for your certificate. - for the catalog
+`catalog_certificate/fullchain.pem`: the certificate file used in most server software.  - for the catalog
+
+Presequisite: 
+- NGINX is installed in the VM.
+
+Steps to run the cloud version:
+
+1. Generate the SSL certificates for app and catalog:
```
Comment (aphethean1) [inline cloud_config/README.md:13]: We will get questions about how.  Adding links to letsencrypt instructions, or copy an paste our normal instructions could be good.

**Review by aphethean1 — COMMENTED** [review]
A few changes to make it clear that this is a script to run.  Looks good otherwise.

File: cloud_config/.env.catalog.cloud [hunk 4]
```diff
@@ -0,0 +1,9 @@
+APP_URL=https://<app_dns_name>
+CATALOG_URL=https://<catalog_dns_name>/api
+CATALOG_DOCBASEURL=https://www.matatika.com/docs
+CATALOG_WEBSITEBASEURL=https://www.matatika.com
+CATALOG_ALLOWED_ORIGINS=https://<app_dns_name>
+CATALOG_AUTH_IDPS_LOCAL_ISSUER_URI=https://<catalog_dns_name>/
+CATALOG_AUTH_IDPS_LOCAL_AUDIENCE=https://<catalog_dns_name>/api
+CATALOG_AUTH_IDPS_PRIMARY_AUDIENCE=https://<catalog_dns_name>/api
+CATALOG_AUTH_IDPS_SECONDARY_AUDIENCE=https://<catalog_dns_name>/api
```
Comment (aphethean1) [inline cloud_config/.env.catalog.cloud:9]: Redundant

File: cloud_config/.env.catalog.cloud [hunk 5]
```diff
@@ -0,0 +1,9 @@
+APP_URL=https://<app_dns_name>
+CATALOG_URL=https://<catalog_dns_name>/api
+CATALOG_DOCBASEURL=https://www.matatika.com/docs
+CATALOG_WEBSITEBASEURL=https://www.matatika.com
+CATALOG_ALLOWED_ORIGINS=https://<app_dns_name>
+CATALOG_AUTH_IDPS_LOCAL_ISSUER_URI=https://<catalog_dns_name>/
+CATALOG_AUTH_IDPS_LOCAL_AUDIENCE=https://<catalog_dns_name>/api
+CATALOG_AUTH_IDPS_PRIMARY_AUDIENCE=https://<catalog_dns_name>/api
```
Comment (aphethean1) [inline cloud_config/.env.catalog.cloud:8]: Probably redundant when local audience is supplied.  These all go into the same list of identify providers

File: cloud_config/.env.app.cloud [hunk 6]
```diff
@@ -0,0 +1,6 @@
+APP_IDENTITY_DOMAIN=matatika.eu.auth0.com
```
Comment (aphethean1) [inline cloud_config/.env.app.cloud:1]: this should be identity.matatika.com (probably means its wrong in CE too)

Comment (gabrielciobotaru77) [reply cloud_config/.env.app.cloud:1]: yes, identity.matatika.com is an alias of matatika.eu.auth0.com

**Pull request merged**